modules/03 Front End Development/Assignments/11/answer.html

```diff
@@ -0,0 +1,42 @@
+<html>
+    <head>
+        <title>
+                Contact Persons
+        </title>
+    </head>
+
+    <body>
+        <table>
+                <tr bgcolor="#FFCCFF">
+                        <th>First Name</th>
+                <th>Last Name</th>
+                <th>Address</th>
+                <th>Phone Number</th>
+                <th>Email Address</th>
+            </tr>
+            <tr bgcolor="#FFCCFF">
+               <td>Daniel</td>
+               <td>Brosh</td>
+               <td>Ashkelon</td>
+               <td>054-5796354</td>
+               <td><a href="mailto:daniel@d.com"> daniel@d.com</a></td> 
+            </tr>
+
+            <tr bgcolor="#FFCCFF">
+                    <td>Nadav</td>
+                <td>Choen</td>
+                <td>Jerusalem</td>
+                <td>052-7855698</td>
+                <td><a href="mailto:nadav@c.com"> nadav@c.com</a></td> 
+            </tr>
+
+            <tr bgcolor="#FFCCFF">
+                    <td>Uriel</td>
+                    <td>Last Name</td>
+                    <td>Haifa</td>
+                    <td>054-8565465</td>
+                    <td><a href="mailto:uriel@u.com"> uriel@u.com</a></td>
+                </tr>
+        </table>
+    </body>
+</html>
```

Comment:
correct. I copy-pasted your answer because I couldn't merge since you added another file to the Pull Request